dev/source/docs/building-setup-windows-cygwin.rst

```diff
@@ -162,4 +164,13 @@ You can run SITL just as you can with any other build environment. In order to s
 
 ::
 
-      export DISPLAY=:0.0 
+      export DISPLAY=:0.0
+      
+Build SITL binaries for Windows
+===============================
+
+You can build 32-bit binaries to be run outside of Cygwin will a small set of companion files. This can be useful when used in conjunction with Mission Planner to run SITL with a stable release build that is not from the master developement branch of ArduPilot.
```

Comment:
```suggestion
You can build 32-bit binaries to be run outside of Cygwin with a small set of companion files. This can be useful when used in conjunction with Mission Planner to run SITL with a stable release build that is not from the master development branch of ArduPilot.
```